ruby_activerecord/boot.rb

```diff
@@ -1,8 +1,101 @@
 require './config/initializers/database'
 require './app/models/shop'
+require './app/models/category'
```

Comment:
you can require all the models dynamically (it will look like autorequire)
something like this
```ruby
Dir['./app/models/*.rb'].each { |f| require f.gsub('.rb', '') }
```